### PR TITLE
added passbolt

### PR DESCRIPTION
**Pull request by AmphoMoronal**

### CODE REVIEW

File: docs/services/passbolt.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      - "/srv/passbolt/postgres:/var/lib/postgresql/data"
+
+  passbolt:
+    image: passbolt/passbolt:latest
+    restart: always
+    tty: true
+    depends_on:
+      - postgres
+    env_file: .passbolt.env
+    environment:
```
Comment (felbinger) [inline docs/services/passbolt.md:26]: please add ports, so it's ready to be used with nginx
```yaml
  ports:
    - "[::1]:8000:80"
```

File: docs/services/passbolt.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      EMAIL_DEFAULT_FROM: "passbolt@domain.de"
+      EMAIL_TRANSPORT_DEFAULT_HOST: "your.mail.host"
+      EMAIL_TRANSPORT_DEFAULT_PORT: 587
+    volumes:
+      - gpg_volume:/srv/passbolt/gpg:/passbolt/gpg
+      - jwt_volume:/srv/passbolt/jwt:/passbolt/jwt
+    command: >
+      bash -c "/usr/bin/wait-for.sh -t 0 postgres:5432 -- /docker-entrypoint.sh"
+
+volumes:
```
Comment (felbinger) [inline docs/services/passbolt.md:39]: We don't use these...

File: docs/services/passbolt.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+      - postgres
+    env_file: .passbolt.env
+    environment:
+      APP_FULL_BASE_URL: https://passbolt.domain.de
+      DATASOURCES_DEFAULT_DRIVER: Cake\Database\Driver\Postgres
+      DATASOURCES_DEFAULT_ENCODING: "utf8"
+      EMAIL_DEFAULT_FROM: "passbolt@domain.de"
+      EMAIL_TRANSPORT_DEFAULT_HOST: "your.mail.host"
+      EMAIL_TRANSPORT_DEFAULT_PORT: 587
+    volumes:
```
Comment (felbinger) [inline docs/services/passbolt.md:33]: We don't use these... transform to directory mounts

File: docs/services/passbolt.md (hunk 1 above)
Comment (AmphoMoronal) [reply docs/services/passbolt.md:26]: The Ports are in the nginx section. Same structure like in paperless

Comment (felbinger) [reply docs/services/passbolt.md:26]: Nope ports are defined in both sections. See https://github.com/felbinger/AdminGuide/blob/main/docs/services/paperless.md?plain=1#L45-L46 for reference

**Pull request closed without merging**